lib/processor/steps/rotate.js

```diff
@@ -42,7 +42,7 @@ module.exports = function(context, stepInfo) {
 
     // remove orientation now that it's been auto-corrected
     // to avoid downloaded asset from being rotated again
-    delete img.info.orientation;
+    // delete img.info.orientation;
```

Comment:
as noted in comments, if you retain orientation in meta, then subsequent image operations will result in incorrect rotations.